shotover-proxy/benches/redis_benches.rs

```diff
@@ -5,111 +5,140 @@ use test_helpers::docker_compose::DockerCompose;
 mod helpers;
 use helpers::ShotoverManager;
 
-fn redis(c: &mut Criterion) {
+fn redis_multi(c: &mut Criterion) {
     let mut group = c.benchmark_group("redis");
     group.throughput(criterion::Throughput::Elements(1));
     group.noise_threshold(2.0);
```

Comment:
If this helps the noise issue then we should drop the noise_threshold down to a suitable value

Reply:
Yeah, I'm thinking around 5-10%?

Reply:
Looking at https://github.com/shotover/shotover-proxy/runs/5104088154?check_suite_focus=true 20% looks about right.
We can bump it up another 5-10% if we get false positives.

Reply:
Alright, changed to 20%.